neo.UnitTests/UT_ProtocolSettings.cs

```diff
@@ -0,0 +1,24 @@
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Diagnostics;
+//using Settings = Neo.Plugins.Settings;
+using ProtocolSettings = Neo.ProtocolSettings;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_ProtocolSettings
+    {
+        [TestInitialize]
+        public void TestSetup()
+        {
+        }
+
+        [TestMethod]
+        public void Test_2_Seconds()
+        {
+            ProtocolSettings.Default.SecondsPerBlock.Should().Be(2);
```

Comment:
Can you make a global constant for the test block time. And compare here that it is equal to the global test block time constant.

Reply:
This is a very good idea :+1:

Reply:
done!